easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.8-10-GCCcore-7.3.0-Ghostscript-9.23-cairo-1.14.12.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '7.0.8-10'
+ghostversion = '9.23'
+cairoversion = '1.14.12'
+versionsuffix = '-Ghostscript-%s-cairo-%s' % (ghostversion, cairoversion)
+
+homepage = 'http://www.imagemagick.org/'
+description = """ImageMagick is a software suite to create, edit, compose, or convert bitmap images"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['http://www.imagemagick.org/download/releases/']
```

Comment:
@crubb Please use `https` (like we already do in existing `ImageMagick` easyconfigs)